tb_plugin/test/test_profiler.py

```diff
@@ -0,0 +1,640 @@
+import json
+import unittest
+
+import tensorboard_plugin_torch_profiler.profiler.trace as trace
```

Comment:
can use relative import?

Reply:
> can use relative import?

Just follow same style of pytorch tests, such as [test_profiler.py](https://github.com/pytorch/pytorch/blob/master/test/test_profiler.py).